apps/studio/prisma/generated/generatedEnums.ts

```diff
@@ -11,6 +11,7 @@ export const ResourceType = {
   CollectionLink: "CollectionLink",
   CollectionPage: "CollectionPage",
   IndexPage: "IndexPage",
+  PageOrder: "PageOrder",
```

Comment:
Could _pages be used for potentially storing other page related metadata? If so, using a more generic name like PageMeta or PageSettings would be good

Reply:
Hmm no, page-related metadata are stored within those individual pages. This is a page ordering that is tied to the folder and not to individual pages.

Reply:
Hmm understand, I just meant if there's potentially other use cases for _pages in future in which using a generic name now will help prevent migrations etc

Reply:
Renamed PageOrder to FolderMeta and `_pages` to `_meta`. I think we can have the option to have a CollectionMeta type in the future to control the ordering of items within the collection, but will need to be careful about the difference between this folder ordering meta vs normal folder metadata.